File: RoQui/src/main/kotlin/dev/joguenco/roqui/ping/controller/PingController.kt
package dev.joguenco.roqui.ping.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class PingController {

    @GetMapping("/ping") fun ping() = Pong()

    class Pong {
        val message = "pong"
    }
}

File: RoQuiApi/RoQui/Ping/PingController.cs
namespace RoQuiApi.RoQui.Ping;

using Microsoft.AspNetCore.Mvc;

[ApiController]
[Route("[controller]")]
public class PingController : ControllerBase
{

    [HttpGet(Name = "GetPing")]
    public Ping Get()
    {
        return new Ping { Message = "Pong" };
    }
}
